infra/gcp/ensure-main-project.sh

```diff
@@ -212,8 +212,11 @@ function empower_cluster_admins_and_users() {
     # TODO(spiffxp): consider replacing this with a service account per namespace
     local prow_deployer_acct prow_deployer_role
     prow_deployer_acct="prow-deployer@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
+    # TODO(spiffxp): use container.deployer once we figure out why it isn't getting RBAC privileges
+    old_prow_deployer_role=$(custom_org_role_name "container.deployer")
```

Comment:
🤦🏻 I somehow saved but then undid and saved the whole point of this change, which was to add `roles/container.admin` and remove `container.deployer`.  This removed/added container.deployer

Fix in https://github.com/kubernetes/k8s.io/pull/2484